dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -1018,6 +1018,65 @@ TRASH_TrashFile(LPCWSTR wszPath)
     return DeleteFileToRecycleBin(wszPath);
 }
 
+static void TRASH_PlayEmptyRecycleBinSound()
+{
+    CRegKey regKey;
+    CHeapPtr<TCHAR> pszValue;
+    CHeapPtr<TCHAR> pszSndPath;
+    DWORD dwType, dwSize;
+    LONG lError;
+
+    /* Open sound key */
+    lError = regKey.Open(HKEY_CURRENT_USER,
+                         _T("AppEvents\\Schemes\\Apps\\Explorer\\EmptyRecycleBin\\.Current"),
```

Comment:
```suggestion
                         L"AppEvents\\Schemes\\Apps\\Explorer\\EmptyRecycleBin\\.Current",
```

Reply:
Is that okay for `CRegKey`?

Reply:
It should, if the whole shell32 is compiled as UNICODE.